src/workers/parser/utils/contains.js

```diff
@@ -7,7 +7,7 @@
 import type { AstLocation, AstPosition } from "../types";
 import type { Node } from "babel-traverse";
 
-function startsBefore(a: AstLocation, b: AstPosition) {
+export function startsBefore(a: AstLocation, b: AstPosition) {
```

Comment:
Reuse the function we did in PR #4381 we can do some optimization in `removeInnerLocations`